packages/ui/src/components/Table/STable.vue

```diff
@@ -148,26 +171,28 @@ const props = withDefaults(
     selectOnIndeterminate: true,
     highlightCurrentRow: false,
     currentRowKey: '',
+    adaptBreakpoint: TABLE_DEFAULT_ADAPT_BREAKPOINT,
+    cardGridBreakpoints: () => TABLE_CARDS_GRID_DEFAULT_BREAKPOINTS,
   },
 )
 
 /* eslint-disable @typescript-eslint/unified-signatures */
 const emit = defineEmits<{
-  (event: 'cell-mouse-enter', ...value: TableCellEventData): void
-  (event: 'cell-mouse-leave', ...value: TableCellEventData): void
-  (event: 'cell-click', ...value: TableCellEventData): void
-  (event: 'cell-dblclick', ...value: TableCellEventData): void
-  (event: 'header-click', ...value: TableHeaderEventData): void
-  (event: 'header-contextmenu', ...value: TableHeaderEventData): void
-  (event: 'row-click', ...value: TableRowEventData): void
-  (event: 'row-dblclick', ...value: TableRowEventData): void
-  (event: 'row-contextmenu', ...value: TableRowEventData): void
-  (event: 'sort-change', value: TableSortEventData): void
-  (event: 'selection-change', value: TableRow[]): void
+  (event: 'mouse-enter:cell', ...value: TableCellEventData): void
+  (event: 'mouse-leave:cell', ...value: TableCellEventData): void
+  (event: 'click:cell', ...value: TableCellEventData): void
+  (event: 'dblclick:cell', ...value: TableCellEventData): void
+  (event: 'click:header', ...value: TableHeaderEventData): void
+  (event: 'contextmenu:header', ...value: TableHeaderEventData): void
+  (event: 'click:row', ...value: TableRowEventData): void
+  (event: 'dblclick:row', ...value: TableRowEventData): void
+  (event: 'contextmenu:row', ...value: TableRowEventData): void
+  (event: 'change:sort', value: TableSortEventData): void
+  (event: 'change:selection', value: TableRow[]): void
```

Comment:
I think it makes more sense to name events like `<namespace>:<event>`, i.e. `cell:mouse-enter`, `cell:mouse-leave`, `cell:click`, `row:click` etc.

It's just like how we name components.

Reply:
We now hame events like `<event>:<namespace>`, not `<namespace>:<event>`

Reply:
Well, that's a dark spot in the style guide. For me, `<event:namespace>` makes less sence.

Reply:
We used it already a lot enough. And there are examples of this style usage like `update:model-value` in vue or event naming in vuetify